frontend/src/components/maps/ParcelPopupView.tsx

```diff
@@ -21,6 +24,18 @@ export const ParcelPopupView = (props: IParcelDetailProps | null) => {
   const parcelDetail: IParcel | null | undefined = props?.parcel;
   const keycloak = useKeycloakWrapper();
   const location = useLocation();
+  const dispatch = useDispatch();
+  const [projectRoute, setProjectRoute] = useState('');
+
+  useEffect(() => {
+    if (parcelDetail?.projectNumber) {
+      (dispatch(fetchProject(parcelDetail?.projectNumber as string)) as any).then(
```

Comment:
When the popup is shown it makes a request to get the appropriate route?

Reply:
Yup, gets the project information (if it project number exists) when the user clicks the marker and sets the SPP-# link to the appropriate route